### ISSUE TITLE
Option to remove _rid and other underscore variables from Document.toString

### ISSUE THREAD

**Issue by kozyr** [Azure/azure-documentdb-java#56]
If I want to query the objects that I stored into DocumentDB, and simply return the JSON back, bunch of internal properties would get returned if I simply return Document.toString for each returned document. Would be nice to have an option to not include them into String serialization.

**Comment by mingaliu** [comment]
Thanks kozyr, we will take your feedback into the consideration in next release.

**Comment by aqua-regia** [comment]
Maybe you can try to deserialize the JSON into java object implements toString() and which does not have fields for getter/setters and attributes for _ related fields and then just use

(gson.fromJson(exampleJSON, examplePOJO.class))toString()

**Comment by kozyr** [comment]
But then I'm doing extra serialization back to JSON from that Java object, and also have to create extra classes. Why not add a method on the Document, giveMeBackWhatIStoredWithoutExtraGarbage()?

**Comment by aqua-regia** [comment]
Yes, that should be there. The thing is there for me this never became a bottleneck. I mostly run a job that does bulk insert and bulk delete and the time taken by them is so much that deserializing to java does not even count in the rate-determining step.

**Comment by kozyr** [comment]
Sure, for offline processing like that it doesn't matter, but what I want to do is return this data back to the clients in real time, in response to their API calls, and I don't want them to see all these extra stuff, nor use the bandwidth for it (nor write extra objects).

**Comment by karlos-oliveira** [comment]
we are on 2020 and nobody solved this?

**Comment by ajay74984** [comment]
any update on it?
